src/popup/components/Header.vue

```diff
@@ -120,7 +120,7 @@ export default {
       return this.$route.meta.hideNotificationsIcon;
     },
     notificationsCount() {
-      return [...this.notifications, ...this.superheroNotifications].filter(
+      return [...(this.notifications || []), ...(this.superheroNotifications || [])].filter(
```

Comment:
I suggest to split this in to 2 lines. I mean the elements added to the array in my opinion should be in separate lines.